packages/@webex/plugin-cc/src/logger-proxy.ts

```diff
@@ -1,9 +1,48 @@
-import {Logger} from './types';
+import {Logger, LogContext, LOGGING_LEVEL} from './types';
+import {LOG_PREFIX} from './constants';
 
 export default class LoggerProxy {
   public static logger: Logger;
 
   public static initialize(logger: Logger): void {
     LoggerProxy.logger = logger;
   }
+
+  public static log(message: string, context: LogContext = {}): void {
```

Comment:
Added custom methods so that we can properly format the logs, we will now include a custom log context which for now houses the method name and the module name(can be expanded).
This is also optional, however we will format it based on a string similar to the one used in `calling`.